README.md

```diff
@@ -1 +1,145 @@
-# Capstone_project_m1
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <br/>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [:movie_camera: Project Presentation](#project-presentation)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Global-Summit] <a name="about-project"></a>
+
+
+**[Global-Summit]** Is a website can be used for confranse or glopbal events and runs smoothly on different devices and screen sizes
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+Built useing Bootstrap and font awesome.
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Responsive design]**
+- **[Hover effects]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Project Presntation -->
+
+## :movie_camera: Project Presentation <a name="project-presentation"></a>
+
+Walking through portfolio outline.
+
+- [Project Presentation Link] [https://www.loom.com/share/5a39affb783a4b66974ec21aa711bd76](https://www.loom.com/share/5a39affb783a4b66974ec21aa711bd76)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link] [[https://jlberbesi.github.io/Microverse-first-module.github.io/](https://jlberbesi.github.io/Capstone_project_m1.github.io/)]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+Use Git or checkout with SVN using the web URL.
+https://github.com/
```

Comment:
- [ ] To help other developers to get your project locally in their machines, it would be a good thing to provide them with this ```git@github.com:jlberbesi/Capstone_project_m1.github.io.git``` so that they can clone it and use it locally. **Note:** The main aim of a README is to document your project so that other people and you know how to setup and use the project when you visit it later.

Reply:
@Nemwel-Boniface  Thanks a lot for this recommendation. I just added it.